client/qiskit_serverless/core/job.py

```diff
@@ -480,39 +441,13 @@ def run_existing(
         arguments: Optional[Dict[str, Any]] = None,
         config: Optional[Configuration] = None,
     ):
-        if isinstance(program, QiskitFunction):
-            title = program.title
-        else:
-            title = str(program)
-
-        tracer = trace.get_tracer("client.tracer")
-        with tracer.start_as_current_span("job.run_existing") as span:
-            span.set_attribute("program", title)
-            span.set_attribute("arguments", str(arguments))
-
-            url = f"{self.host}/api/{self.version}/programs/run_existing/"
-
-            data = {
-                "title": title,
-                "arguments": json.dumps(arguments or {}, cls=QiskitObjectsEncoder),
-            }  # type: Dict[str, Any]
-            if config:
-                data["config"] = asdict(config)
-            else:
-                data["config"] = asdict(Configuration())
-
-            response_data = safe_json_request(
-                request=lambda: requests.post(
-                    url=url,
-                    json=data,
-                    headers={"Authorization": f"Bearer {self._token}"},
-                    timeout=REQUESTS_TIMEOUT,
-                )
-            )
-            job_id = response_data.get("id")
-            span.set_attribute("job.id", job_id)
-
-        return Job(job_id, job_client=self)
+        warnings.warn(
```

Comment:
d should not have been used by the user directly. Can we just take out?

Reply:
Take out the warning do you mean, @akihikokuroda ?

Reply:
I wonder if we can take out the method.

Reply:
I'm not sure, let me check 👍

Reply:
Is this what you were expenting, @akihikokuroda ? [2363e98](https://github.com/Qiskit/qiskit-serverless/pull/1320/commits/2363e98f01d00969e3bae867582c3ef9dda7373e)

Reply:
Yes. Thanks!